Makefile

```diff
@@ -36,7 +36,7 @@ test-e2e-short:
 # run only happy path E2E tests with cometmock
 # this set of traces does not test equivocation but it does check downtime
 test-e2e-short-cometmock:
-	go run ./tests/e2e/... --short-happy-path --use-cometmock --use-gorelayer
+	go run ./tests/e2e/... --cometmock-happy-path --use-cometmock --use-gorelayer
```

Comment:
I think all of this is great and very exciting.

However a small suggestion, maybe we refactor the `main.go` flag parsing so that `--cometmock-happy-path` uses go relayer by default so the flag does not need to be specified.

The flag could just be `-cometmock-happy-path --use-cometmock`.

This can also be done as part of a larger refactor, I'm just asking for opinions

Reply:
To me it seems implicit that if you're specifying `--cometmock-happy-path`, both `--use-cometmock` and `--use-gorelayer` would default to true

Reply:
pls note that PR #1191 is changing the way tests can be selected.
Keeping selection of tests and infra seperated regarding parameters is better from a usablity perspective (full flexibility with reduced set of option).
I'd be in favor in not renaming the 'test case' and rather not having 'cometmock' in the name as it leads to assumptions that infra/test setup is using cometmock which is not the case. Running  --short-happy-path --use-cometmock (with or without implicit usage of go-relayer) is clearer IMO.

Reply:
> assumptions that infra/test setup is using cometmock which is not the case. Running --short-happy-path --use-cometmock (with or without implicit usage of --use-gorelayer) is clearer IMO.

I think this is a good approach to take.